regression/goto-analyzer-simplify/simplify-multiply-by-zero/test.desc

```diff
@@ -0,0 +1,5 @@
+CORE
+main.c
+"--variable"
+
+main#return_value = 0;
```

Comment:
I'd say that all tests should at bare minimum check the exit code and signal.

Reply:
I can do that, but none of the other tests in goto-analyzer-simplify do this.

Reply:
Then maybe they should start doing so: tests outside that directory routinely do so.

Reply:
OK, I will add the tests for that. I also just noticed that I broke some other regression tests elsewhere with this change, so this is on hold until I figure out what's going on there.